Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationPage.xaml.cs

```diff
@@ -19,52 +19,52 @@ namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
     public sealed partial class SystemInformationPage : Page
     {
         // To get application's name:
-        public string ApplicationName => SystemInformation.ApplicationName;
+        public string ApplicationName => App.SystemInformation.ApplicationName;
```

Comment:
Can't we just use x:Bind directly from XAML, without the need for all these proxy properties in code behind?

Reply:
We can. ` 
By default, I didn't change the sample code. We were already able to use `x:Bind` before the change.

Reply:
Oh, absolutely, you're right about that 👍
I was suggesting that given that you're already making a PR editing this area, maybe it'd be a good occasion to also make that change in the sample app, so that it'd also be helpful to showcase the new `x:Bind` to function feature to new developers? Or if not, if we prefer to keep this PR smaller in scope, it could be a good idea to make a note of this for the future 😊

Reply:
Let's do this in another PR.

Reply:
Sounds good, thank you for the info, and nice job again with this PR! 👍